src/integrationTest/java/org/opensearch/test/framework/cluster/MinimumSecuritySettingsSupplierFactory.java

```diff
@@ -70,6 +70,7 @@ private Settings.Builder minimumOpenSearchSettingsBuilder(int node, boolean sslO
 			builder.putList("plugins.security.authcz.admin_dn", testCertificates.getAdminDNs());
 			builder.put("plugins.security.compliance.salt", "1234567890123456");
 			builder.put("plugins.security.audit.type", "noop");
+			builder.put("plugins.security.background_init_if_securityindex_not_exist", "false");
```

Comment:
for my knowledge, what does this setting do?

Reply:
This configuration parameter is used in method `org.opensearch.security.configuration.ConfigurationRepository#initOnNodeStart`. If the value is `true ` then the thread defined in the constructor `org.opensearch.security.configuration.ConfigurationRepository#ConfigurationRepository`is started. The thread is responsible for:

1. Loading default configuration (if another configuration parameter `plugins.security.allow_default_init_securityindex` is equal to `true`, but the default is `false`)
2. Loading security plugin configuration from index `.opendistro_security` to local node cache.

Both operations are not needed during the tests because in most cases a new cluster is created, security plugin configuration is inserted into index `.opendistro_security` and then a configuration update request is sent to the cluster, please see method `org.opensearch.test.framework.TestSecurityConfig#initIndex` 

In practice switching the parameter `plugins.security.background_init_if_securityindex_not_exist` to `true` causes logging additional WARN messages.

@nibix please correct me if I went wrong.